File: app/src/main/java/de/chrisward/theyworkforyou/helper/ResourceHelper.java
package de.chrisward.theyworkforyou.helper;

import de.chrisward.theyworkforyou.R;

public class ResourceHelper {
    public static int getPartyColor(String party) {
        switch(party) {
            case "Conservative":
                return R.color.conservative;
            case "DUP":
                return R.color.dup;
            case "Green":
                return R.color.green;
            case "Labour":
            case "Labour/Co-operative":
                return R.color.labour;
            case "Liberal Democrat":
                return R.color.libdem;
            case "Plaid Cymru":
                return R.color.plaid_cymru;
            case "Scottish National Party":
                return R.color.snp;
            case "Sinn Féin":
                return R.color.sinn_fein;
            case "Speaker":
                return R.color.speaker;
            default:
                return R.color.independent;
        }
    }
}

File: app/src/main/java/de/chrisward/theyworkforyou/helper/ResourceHelper.kt
package de.chrisward.theyworkforyou.helper

import de.chrisward.theyworkforyou.R

object ResourceHelper {
    fun getPartyColor(party: String): Int {
        when (party) {
            "Conservative" -> return R.color.conservative
            "DUP" -> return R.color.dup
            "Green" -> return R.color.green
            "Labour", "Labour/Co-operative" -> return R.color.labour
            "Liberal Democrat" -> return R.color.libdem
            "Plaid Cymru" -> return R.color.plaid_cymru
            "Scottish National Party" -> return R.color.snp
            "Sinn Féin" -> return R.color.sinn_fein
            "Speaker" -> return R.color.speaker
            else -> return R.color.independent
        }
    }
}
